### PR TITLE
Update beats to bd6981d

### PR DESCRIPTION
**Pull request by ruflin**
Important Changes / Additions:
* Add support for cloud id. See https://github.com/elastic/beats/pull/4964 for more details. This could simplify the onboard for users that use apm-server with the Cloud in the future.
* Be able to overwrite index pattern in the index template. See https://github.com/elastic/beats/pull/4769 for more details. This allows to clean up our system tests.

### CODE REVIEW

File: apm-server.reference.yml [hunk 1]
```diff
@@ -617,10 +631,14 @@ output.elasticsearch:
 # Set to false to disable template loading.
 #setup.template.enabled: true
 
-# Template name. By default the template name is apm-server.
-# The version of the beat will always be appended to the given name
-# so the final name is apm-server-%{[beat.version]}.
-#setup.template.name: "apm-server"
+# Template name. By default the template name is "apm-server-%{[beat.version]}"
```
Comment (simitt) [inline apm-server.reference.yml]: should we keep havig. `beat.version` here or change to `apmserver.version`?

File: apm-server.reference.yml [hunk 2]
```diff
@@ -617,10 +631,14 @@ output.elasticsearch:
...
+# The template name and pattern has to be set in case the elasticsearch index pattern is modified.
+#setup.template.name: "apm-server-%{[beat.version]}"
+
+# Template patttern. By default the template patter is "-%{[beat.version]}-*" to apply to the default index settings.
```
Comment (ruflin) [inline apm-server.reference.yml]: That is a typo I need to fix on the beats side.

File: apm-server.reference.yml (hunk 1 above)
Comment (ruflin) [inline apm-server.reference.yml]: The fields used here must be part of the event. We have `beat.version` but we don't have `apm-server.version` in our events.

**Comment by ruflin** [comment]
@simitt For updating _beats: Yes

Our use case for overwriting the index pattern name is for testing. On the beats side especially for logging people create their own templates. But in general it's only expert users that do this and I would not expect it to happen on the apm-server side at first.

**Pull request merged**